jest.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  globals: {
+    'ts-jest': {
+      tsConfig: 'tsconfig.json',
+    },
+  },
+  moduleFileExtensions: ['ts', 'js'],
+  transform: {
+    '^.+\\.(ts|tsx)$': './node_modules/ts-jest',
+  },
+  testMatch: ['**/tests/**/*.test.(ts|js)'],
+  testEnvironment: 'node',
+  preset: 'ts-jest', // use this if you are using TypeScript
+  // globalTeardown: './jest.global-teardown.js' // optional: will be called once after all tests are executed
```

Comment:
do we need to keep this commented-out code?

Reply:
Not really, I'll remove it!